tinytuya/scanner.py

```diff
@@ -944,7 +944,9 @@ def _generate_ip(networks, verbose, term):
         for addr in ipaddress.IPv4Network(network):
             yield str(addr)
 
-def _print_device_info( result, note, term, extra_message=None ):
+def _print_device_info( result, note, term, extra_message=None, verbose=True ):
+    if not verbose:
+        return
```

Comment:
I noticed while testing that `verbose=False` was not honored.

```python
scanner.devices(verbose=False, color=False, poll=False, forcescan="10.0.0.0/24",discover=False)
```

Printed all the devices.  I updated scanner to ignore prints if verbose=False.